src/Admin/AdminInterface.php

```diff
@@ -659,14 +650,28 @@ public function getListMode();
      */
     // public function isCurrentRoute(string $name, ?string $adminCode = null): bool;
 
+    /**
+     * @param string $mode
+     */
+    public function setListMode($mode);
+
+    /**
+     * return the list mode.
+     *
+     * @return string
+     */
+    public function getListMode();
+
     /*
      * Configure buttons for an action
      */
-    // public function configureActionButtons(string $action, ?object $object = null): void;
+    // public function configureActionButtons(string $action, ?object $object = null): array;
```

Comment:
Is the return type change intended?

Reply:
Yes, the method return the new action list